src/plugins/persistence/couch/CouchObjectProvider.js

```diff
@@ -557,11 +556,31 @@ class CouchObjectProvider {
     update(model) {
         let intermediateResponse = this.getIntermediateResponse();
         const key = model.identifier.key;
+        model = this.toPersistableModel(model);
```

Comment:
Were there issues where the model was being modified?

Reply:
The only thing this line does is to strip identifiers from objects before they are persisted. Persisting our identifiers in Couch DB is redundant at best, and at worst could become stale. The `key` part of the identifier is just duplicating what we use to identify the document in Couch already, and the namespace is irrelevant (for the purposes of Couch DB - it's an Open MCT concept).